packages/frontend/src/app/utils/index.ts

```diff
@@ -50,7 +37,7 @@ export const getPostSummaries = (posts: any[]): PostSummary[] => {
       category: subSubcategory?.subcategory?.name,
       subSubcategory: subSubcategory?.name,
       publishedDate: post.publishedDate,
-      theme: getThemeFromCategory(subSubcategory?.subcategory?.category?.slug),
+      theme: subSubcategory?.subcategory?.category?.themeColor,
```

Comment:
we may need fallback `DEFAULT_THEME_COLOR` here in case the query is missing `themeColor` field